### DISCUSSION TITLE
64Drive HW1?

### DISCUSSION THREAD

**Discussion by supavgx** [Polprzewodnikowy/N64FlashcartMenu#48] [General]
Im just wondering if its compatible with HW1 of the 64drive since its not clear what hardware revision it works with.

**Comment by Polprzewodnikowy** [comment]
Menu should work with both hardware revisions. However, replacing `menu.bin` with this menu was not tested in practice. I only tested 64drive integration in modified emulator (ares). Would be great to hear if this works on real hardware.
